### DISCUSSION TITLE
Welcome to winget-pkgs-index Discussions!

### DISCUSSION THREAD

**Discussion by svrooij** [svrooij/winget-pkgs-index#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by brandonall** [comment]
Do you have plans to increase your package selection to msstore apps that use number and letter strings as ID. An example would be Microsoft's company portal, which can be installed with winget but not with wintuner.
